test/src/test/java/test/org/dst/client/StringProxyTest.java

```diff
@@ -0,0 +1,34 @@
+package test.org.dst.client;
+
+
+import junit.framework.Assert;
+import org.dst.client.DefaultDstClient;
+import org.dst.client.DstClient;
+import org.dst.client.exception.KeyNotFoundException;
+import org.junit.jupiter.api.Test;
+
+
+public class StringProxyTest {
+
+    private final static String serverAddress = "";
```

Comment:
we need to get all configurations in `dst.conf`

Reply:
Thanks. But now we have not conf file.